tools/packaging/common/istio-iptables.sh

```diff
@@ -410,7 +410,7 @@ fi
 if [ -z "${DISABLE_REDIRECTION_ON_LOCAL_LOOPBACK-}" ]; then
   # Redirect app calls back to itself via Envoy when using the service VIP or endpoint
   # address, e.g. appN => Envoy (client) => Envoy (server) => appN.
-  iptables -t nat -A ISTIO_OUTPUT -o lo ! -d 127.0.0.1/32 -j ISTIO_REDIRECT
```

Comment:
Did you test that the loop is gone ? I don't see how. IN_REDIRECT will also go to a 'original dst', problem is still there.

Reply:
See https://github.com/istio/istio/pull/15710#discussion_r306431097
Short answer is no. The changes here is to fix another loop